medical_base_history/models/medical_history_entry.py

```diff
@@ -46,15 +47,15 @@ class MedicalHistoryEntry(models.Model):
         help='Copy of old record for history auditing',
         readonly=True,
         store=True,
-        select=False,
+        index=False,
         compute='_compute_old_record_dict',
         inverse='_write_old_record_dict',
     )
     new_record_dict = fields.Text(
```

Comment:
We can use `fields.Serialized` here to achieve the same effect here & reduce some logic. Pretty sure I didn't know that field type existed when I built this.